openid-federation-1_0.xml

```diff
@@ -1378,77 +1378,43 @@
 	      to achieve request authentication.
 	      They are the subset of those listed in the
 	      IANA "OAuth Token Endpoint Authentication Methods" registry <xref target="IANA.OAuth.Parameters"/>
 	      that perform client authentication by proving possession of a private key.
-              These client authentication methods are:
+	      The <spanx style="verb">pushed_authentication_methods_supported</spanx>
+	      metadata value lists the client authentication methods
+	      supported by the OP that can be used in this manner.
+	      Its value is a JSON array of supported client authentication methods.
+	      Values that MAY be used are:
               <list style="hanging">
                 <t hangText="private_key_jwt">
                   <vspace/>
                   This client authentication method is described in Section 9 of
                   <xref target="OpenID.Core">OpenID Connect Core 1.0</xref>.
-                  Note that if <spanx style="verb">private_key_jwt</spanx> is
-                  used, the audience of the signed JWT MUST be either the URL of
-                  the Authorization Server's Authorization Endpoint or the
-                  Authorization Server's Entity Identifier.
-                </t>
-                <t hangText="tls_client_auth">
-                  <vspace/>
-                  Section 2.1 of
-                  <xref target="RFC8705"/>.
+                  When <spanx style="verb">private_key_jwt</spanx> is used,
+		  the audience of the signed JWT MUST be
+                  the Authorization Server's Entity Identifier.
                 </t>
                 <t hangText="self_signed_tls_client_auth">
                   <vspace/>
-                  Section 2.2 of
+                  This client authentication method is described in Section 2.2 of
                   <xref target="RFC8705"/>.
                 </t>
               </list>
-              <vspace blankLine="1"/>
-
-              A request authentication method that is not a client authentication method is:
-              <list style="hanging">
-                <t hangText="request_object">
-                  <vspace/>
-                  Use of a Request Object, as described in
-                  <xref target="OpenID.Core">OpenID Connect Core 1.0</xref>.
-		  The request authentication is performed by verifying that
-		  the request object is signed with a key that the RP controls.
-		  Use of Request Objects by Automatic Registration is described in
-                  <xref target="automatic"/>.
-                </t>
-              </list>
-	      The <spanx style="verb">request_object</spanx> method MAY be used at either endpoint.
-              <vspace blankLine="1"/>
-
-	      Additional request authentication methods MAY be defined and used.
+	      <vspace blankLine="1"/>
+	      In both cases, the signing key MUST be one of the keys
+	      in the RP's JWK Set published in its metadata.
```

Comment:
It may be helpful to clarify that the keys referenced here are OIDC RP keys (i.e., published under `metadata`->`openid_relying_party`, and NOT Federation keys).